src/solver/timeStepper.cpp

```diff
@@ -57,6 +57,7 @@ ablate::solver::TimeStepper::TimeStepper(std::string nameIn, std::shared_ptr<abl
 ablate::solver::TimeStepper::~TimeStepper() { TSDestroy(&ts) >> checkError; }
 
 void ablate::solver::TimeStepper::Initialize() {
+    StartEvent((this->name + "::Initialize").c_str());
```

Comment:
Will this duplicate the individual Initialize logs in the child classes or is this only acting on the domain setups?

Reply:
This will only act on the highest level timestpper init.  If any other solver records an init it will be under this one.